src/endpoints/getResults.ts

```diff
@@ -6,19 +6,35 @@ import { popSlashSource } from '../utils/parsing'
 import { HLTVConfig } from '../config'
 import { fetchPage, toArray, getMatchFormatAndMap } from '../utils/mappers'
 
-export const getResults = (config: HLTVConfig) => async ({ pages = 1 } = {}): Promise<
+export const getResults = (config: HLTVConfig) => async ({ pages = 1, eventId = 0 } = {}): Promise<
```

Comment:
Why is the default to 0 needed here?

Reply:
Yeah it's not... I'll remove it.

Reply:
Did you mean I should set it to `undefined`?

Reply:
I cheated and looked at how emha did it for teamId in his PR and made the same changes :)
I'm still new to typescript and didn't know how to remove the default value...